mathics/core/evaluation.py

```diff
@@ -379,7 +379,17 @@ def get_stored_result(self, eval_result, output_forms):
     def stop(self) -> None:
         self.stopped = True
 
-    def format_output(self, expr, format=None):
+    def format_output(
```

Comment:
probably, `format_output` should be a parameter of `Evaluation.evaluate`, or in the `Evaluation.__init__` method.

Reply:
Not sure about that, but now this is used in all the backends, so let's keep for a while